### PR TITLE
EIP-7549: Attestation packing

### PR DESCRIPTION
**Pull request by rkapka**
**What type of PR is this?**

Feature

**What does this PR do? Why is it needed?**

Implementation of `compute_on_chain_aggregate` from https://github.com/ethereum/consensus-specs/blob/dev/specs/electra/validator.md#attestations. This is a naive version that takes the top aggregate for each attestation_data+committee combination.

### CODE REVIEW

File: beacon-chain/rpc/prysm/v1alpha1/validator/proposer_attestations.go [hunk 1]
```diff
@@ -66,23 +66,43 @@ func (vs *Server) packAttestations(ctx context.Context, latestState state.Beacon
...
 		as, err := attaggregation.Aggregate(as)
 		if err != nil {
 			return nil, err
 		}
-		attsForInclusion = append(attsForInclusion, as...)
+		attsById[id] = as
+	}
+
+	var attsForInclusion proposerAtts
+	if postElectra {
```
[the remark this answers is not in the document]
Comment (james-prysm) [reply beacon-chain/rpc/prysm/v1alpha1/validator/proposer_attestations.go:87]: I noticed postElectra := slots.ToEpoch(blkSlot) >= params.BeaconConfig().ElectraForkEpoch

**Review by james-prysm — APPROVED** [review]
please add a test for edge cases for post electra checks in a subsequent pr

**Review by james-prysm — APPROVED** [review]
nice thanks for the comment
